aioodbc/pool.py

```diff
@@ -135,6 +136,9 @@ def acquire(self):
             try:
                 conn = await connect(echo=self._echo, loop=self._loop,
                                      **self._conn_kwargs)
+                coro = self._posthook(conn._conn)
+                if asyncio.iscoroutine(coro):
```

Comment:
I think hook should be always coroutine

Reply:
I do not think this is good place to add hook here, it should be bounded to connection not pool

Reply:
🤔 indeed.